app-accessibility/flite/flite-2.1.recipe

```diff
@@ -0,0 +1,143 @@
+SUMMARY="A light-weight text-to-speech engine"
+DESCRIPTION="Flite is a text-to-speech engine developed by the Carnegie \
+Mellon University. As a light-weight alternative to the Festival engine, it \
+is desinged to fit into small embedded machines as well as large-scale \
+servers. In addition to its native format flite supports voices generated by \
+the FestVox project as well."
+HOMEPAGE="http://www.festvox.org/flite/
+	https://github.com/festvox/flite"
+COPYRIGHT="1986 University of Toronto
+	1992, 1995 Markus Mummert
+	1996-2004 University of Edinburgh, UK
+	1999-2014 Language Technologies Institute, Carnegie Mellon University
+	2001 Cepstral, LLC
+	2001 Sun Microsystems, Inc.
+	2003-2004 Nagoya Institute of Technology
+	2003-2004 Nara Institute of Science and Technology
+	2008-2012 University of Illinois at Urbana-Champaign"
+# Although individually many files have different licenses, as a whole this
+# license covers all the files.
+LICENSE="Flite"
+REVISION="1"
+SOURCE_URI="http://www.festvox.org/flite/packed/flite-${portVersion}/flite-${portVersion}-release.tar.bz2"
+CHECKSUM_SHA256="c73c3f6a2ea764977d6eaf0a287722d1e2066b4697088c552e342c790f3d2b85"
+SOURCE_DIR="flite-${portVersion}-release"
+PATCHES="flite-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="2.1"
```

Comment:
Maybe? (saves changes for future bump):
```diff
- libVersion="2.1"
+ libVersion="$portVersion"
```

Reply:
I'm going to change this, although I can't be too sure about it due to the fact that this may actually mess up version bumps. (example: minor change bumps are much less likely to have their entire library versions bumped as well)

Reply:
Looking at the previous recipe it used libVersion = portVersion also (2.0.0), so I guess we can assume they will stick to that